portfolio/src/main/webapp/community.html

```diff
@@ -0,0 +1,30 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <title>Community Page</title>
+    <script src="community.js"></script>
+    <link rel="stylesheet" href="community.css">
+  </head>
+  <body>
+    <div id="community-content">
+        <div id="comment-form-section">
+            <h2>Here are all the wonderful things people have to say about this website</h2>
+            <form action="/comments" method="POST">
+                <p>Username:</p>
+                <input name="username" />
+
+                <p>Comment</p>
+                <input name="comment" />
+                <br />
+
+                <input type="submit" />
+            </form>
+        </div>
+
+        <!-- Comments will be loaded in this div -->
```

Comment:
I'd start to write these HTML comments as TODOs as well.. it makes it easier to consistently notice where there is remaining work to be done.  So something like "TODO: Load comments in this div"

Reply:
I don't think this is a TODO. But I can see how gabe interpreted it that way. Maybe something more like "These divs are populated with comments by the JS"

Reply:
Unresolving. Please update.

Reply:
Oh I see now.  Yea my bad.  I was in looking-for-todo-mode and would be fine with what is there or with what Kenzie said =)

Reply:
Resolved